OracleWebLogic/samples/12212-msiserver/container-scripts/launcher.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
```

Comment:
Hi Avi,

By handlers for stop, did you mean something like trapping (using trap) SIGKILL and SIGTERM and passing them on to server process?

What do you mean by a handler for start? Isn't launcher the script that will run when we start the container?